exercises/practice/largest-series-product/example.cpp

```diff
@@ -0,0 +1,36 @@
+#include "example.h"
+#include <numeric>
+#include <algorithm>
+#include <stdexcept>
+#include <regex>
+namespace largest_series_product {
+
+    int largest_product(const std::string& series, int size) {
```

Comment:
Here and in the `.h` file the parameters are named `series` and `size`. In the canonical-data.json the parameters are named `digits` and `span`, I'd suggest keeping the names in the example solution (BTW: the error messages use the names `digits` and `span`).